docs/guides/on-demand.mdx

```diff
@@ -99,27 +99,27 @@ otherwise
 
 > **Note:** using the `GET /api/task/$TASK_ID` request provides different information depending on whether the asset is _imported_ or _exported_ .
 
-### For imported assets
+### Assets uploaded with URL
 
 - Information on video specs such as codec and formats are present.
 
-### For assets exported to IPFS
+### Assets stored on IPFS
 
-- The response will provide information such as the `metadat` and `CID`.
+- The response will provide information such as the `metadata` and `CID`.
 
 ## Methods for Uploading Video
 
 There are two methods to upload a video to Livepeer:
 
-- Upload via URL
-- Upload locally with an `.mp4` file format
+- Upload with URL
+- Upload from local storage with an `.mp4` file format
 
-## Upload via URL
+## Upload with URL
 
-When using the upload via URL method:
+When using the upload with URL method:
 
 - Provide the name of the asset
-- Provide the URL of the asset that should be imported
+- Provide the URL of the asset that should be uploaded
```

Comment:
Even though we are calling the overall process "upload", I believe we should refrain from using that word in sentences where it wouldn't make sense, like here, to avoid confusion about the API. 

In the "upload with URL", an asset is created in the VOD API which we are calling an "upload", but the asset is not "uploaded" per se, it's actually **downloaded** from the provided URL. We should avoid the term "download" anyway or it could get very confusing, so I'd say we just avoid using any terms related to "down/uploading" altogether when describing the "upload with URL".

To give some concrete suggestion, I'd go with sth like this, wdyt?:
```suggestion
- Provide the URL where to find the content for the new asset
```

Reply:
Yes, I agree now that both are referred to as `upload`